store/models/run.go

```diff
@@ -142,6 +145,20 @@ func JobRunsWithStatus(runs []JobRun, status RunStatus) []JobRun {
 	return rval
 }
 
+// RunRequest stores the fields used to initiate the parent job run.
+type RunRequest struct {
+	ID        uint `gorm:"primary_key"`
+	RequestID *null.String
+	TxHash    *common.Hash
+	Requester *common.Address
+	CreatedAt time.Time
+}
+
+// NewRunRequest returns a new RunRequest instance.
+func NewRunRequest() RunRequest {
```

Comment:
`New` usually means "returns a pointer" in go. `Make` is usually used for copies.

Reply:
https://www.godesignpatterns.com/2014/04/new-vs-make.html

and just most of the go codebases I've read through